prod/westeurope/internal/api/functions_admin/function_app/terragrunt.hcl

```diff
@@ -72,6 +85,15 @@ inputs = {
     AZURE_APIM                = "io-p-apim-api"
     AZURE_APIM_HOST           = "api-internal.io.italia.it"
     AZURE_APIM_RESOURCE_GROUP = "io-p-rg-internal"
+
+    MessageContentStorageConnection = dependency.storage_account.outputs.primary_connection_string
```

Comment:
You have not run terragrunt hclfmt - the lint equivalent in other languages....I'll do that.

Reply:
If I do, it changes 27 files 😅